docs/en/latest/practices/enable-authentication-and-restriction.md

```diff
@@ -314,10 +516,10 @@ EOF
 
 **Example usage**
 
-Requests from jack1:
+* Requests from jack1:

```

Comment:
Just to make sure: whether the `*` signal is added by your IDE?

Reply:
Not, I added it.